src/clusterfuzz/_internal/bot/tasks/utasks/corpus_pruning_task.py

```diff
@@ -642,7 +662,7 @@ def do_corpus_pruning(context, revision):
 
   # Store corpus stats into CoverageInformation entity.
   project_qualified_name = context.fuzz_target.project_qualified_name()
-  today = datetime.datetime.utcnow().date()
+  today = datetime.datetime.utcnow()
```

Comment:
Why are we making this change?

Reply:
It is easier to convert a datetime to a proto timestamp as we have `timestamp_pb2.Timestamp.FromDateTime()`.
This way I do the conversion to proto in `_extract_coverage_information` in `utask_main`.

And then call `cov_info.timestamp.ToDatetime().date()` from `_save_coverage_information` in `utask_postprocess` to get the date part only as before.

Reply:
Aren't we loosing information in the output compared to what we had before here?

```py
from google.protobuf.timestamp_pb2 import Timestamp
import datetime
now = datetime.datetime.now(datetime.timezone.utc)
timestamp = Timestamp()
timestamp.FromDatetime(now)
deserialize = timestamp.ToDatetime().date()
```

This at least seems to work as expected on my machine.

Reply:
This is what I am doing currently. I call `.ToDatetime().date()` on the deserialized timestamp in postprocess.

Reply:
`today` is set in `utask_main`. Instead of extracting the datepart directly there, I serialize it and extract the date part from the deserialized timestamp.

Reply:
Ok, sgtm!